pkg/systemd/overview-cards/cryptoPolicies.jsx

```diff
@@ -107,7 +107,7 @@ const CryptoPolicyDialog = ({
         DEFAULT: _("Recommended, secure settings for current threat models."),
         "DEFAULT:SHA1": _("DEFAULT with SHA-1 signature verification allowed."),
         LEGACY: _("Higher interoperability at the cost of an increased attack surface."),
-        "LEGACY:AD-SUPPORT": _("LEGACY with Active Directory interoperability."),
+        "Legacy:ad-support": _("LEGACY with Active Directory interoperability."),
```

Comment:
Please no -- we said that the *only* string mangling is "DEFAULT" → `_("Default")`. This should stay as-is.